charactersheet/charactersheet/services/persistence_service.js

```diff
@@ -131,6 +131,42 @@ PersistenceService.findBy = function(model, property, value) {
 };
 
 /**
+ * Given a model class, return all of the stored instances that match a given
+ * array of filter criteria.
+ *
+ * Parameters
+ * ----------
+ *
+ * model: The prototype for the type of model that is being searched for.
+ * properties: A static array of properties that the data object will be 
+ * compared against.
+ *
+ * Returns
+ * -------
+ *
+ * A list of objects of the desired type, which match the list of criteria.
+ *
+ * Usage
+ * -----
+ * ```javascript
+ *         var status = PersistenceService.findByListOfProperties(Status,
+ *           [{'name': 'characterId', 'value': key},
```

Comment:
I'd prefer if this was `key` not `name`. It's more conventional and consistent with existing calls.